BAZProject/DetailsMovies/DetailsCollectionViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  DetailsCollectionViewCell.swift
+//  BAZProject
+//
+//  Created by Manuel S on 27/04/23.
+//
+
+import UIKit
+
+class DetailsCollectionViewCell: UICollectionViewCell {
+    
```

Comment:
Un Mark aquí funcionaría bien:
```suggestion
    // MARK: - IBOutlets
```